lib/galaxy/datatypes/binary.py

```diff
@@ -657,7 +657,8 @@ def set_meta(self, dataset, overwrite=True, **kwd):
                                        '__dataset_%d_%s' % (dataset.id, os.path.basename(index_file.file_name)))
         os.symlink(dataset.file_name, dataset_symlink)
         try:
-            pysam.bcftools.index(dataset_symlink)
+            cmd = ['python', '-c', "import pysam.bcftools; pysam.bcftools.index('%s')" % (dataset_symlink)]
+            subprocess.check_call(cmd)
```

Comment:
This is not needed any more after commit e561267bfc6e36853b4f56ebabda7acbdfa9eeb, right?